assets/brunch-config.js

```diff
@@ -47,7 +47,8 @@ exports.config = {
     babel: {
       // Do not use ES6 compiler in vendor code
       ignore: [/vendor/],
-      presets: ['env', 'react']
+      presets: ['env', 'react'],
+      plugins: ['transform-class-properties', 'transform-object-rest-spread'],
```

Comment:
This is to support some other modern javascript syntax that isn't included with the default 'env' preset
https://babeljs.io/docs/plugins/transform-class-properties/
and
https://babeljs.io/docs/plugins/transform-object-rest-spread/